doonut-app-external-api/src/main/kotlin/com/doonutmate/calendar/service/CalendarService.kt

```diff
@@ -0,0 +1,34 @@
+package com.doonutmate.calendar.service
+
+import com.doonutmate.calendar.controller.dto.CalendarResponse
+import com.doonutmate.calendar.controller.dto.CalendarResult
+import com.doonutmate.doonut.calendar.service.CalendarBusinessService
+import org.springframework.data.domain.Pageable
+import org.springframework.stereotype.Service
+import java.time.Instant
+
+@Service
+class CalendarService(
+    private val calendarBusinessService: CalendarBusinessService,
+    private val calendarFacadeService: CalendarFacadeService,
+) {
+    fun get(id: Long?, time: Instant?, page: Pageable): CalendarResult<CalendarResponse> {
+        val calendarList: List<CalendarResponse> = getBoards(id, time, page)
+        val lastIdOfList: Long? = if (calendarList.isEmpty()) null else calendarList[calendarList.size - 1].id
+        val lastTimeOfList: Instant? = if (calendarList.isEmpty()) null else calendarList[calendarList.size - 1].updatedAt
```

Comment:
[lastOrNull()](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/last-or-null.html)을 사용하면 좀 더 kotlin다운 코드가 될 것 같아요.

```suggestion
        val lastIdOfList: Long? = calendarList.lastOrNull()?.id
        val lastTimeOfList: Instant? = calendarList.lastOrNull()?.updatedAt
```